recipes/conversation_view/attachment_carousel/attachment_carousel.vue

```diff
@@ -0,0 +1,263 @@
+<template>
+  <div
+    class="dt-attachment-carousel"
+    role="presentation"
+    @mouseenter="showCarouselArrows = true"
+    @mouseleave="showCarouselArrows = false"
+    @focusin="showCarouselArrows = true"
+    @focusout="showCarouselArrows = false"
+  >
+    <div
+      v-if="mediaList.length > 0"
+      ref="carousel"
+      class="dt-attachment-carousel--media-list"
+      :aria-label="attachmentAriaLabel"
+      @scroll="handleScroll"
+    >
+      <!-- Image list -->
+      <div
+        v-for="(mediaItem, index) in mediaList"
+        :key="`media-${index}`"
+        role="presentation"
+        class="d-ps-relative"
+        @focusin="closeButton(true, index)"
+        @focusout="closeButton(false, index)"
+        @mouseenter="closeButton(true, index)"
+        @mouseleave="closeButton(false, index)"
+      >
```

Comment:
Hmm something seems messed with keyboard functionality. Tabbing between them makes the focus go to some "hidden" place before going to the next item. Maybe it is going to the close button but it's invisible if it is.

Reply:
Yea this is something i wasnt able to fix. Still trying. It doesnt tab to the close button. Coz i kept a focus listener and it didnt get triggered. But i do see this issue.

Reply:
solved using opacity to show hide buttons